grr/server/grr_response_server/gui/ui/components/flow_args_form/osquery_form.ts

```diff
@@ -46,11 +64,43 @@ export class OsqueryForm extends FlowArgumentForm<OsqueryArgs> implements
   }
 
   ngOnInit(): void {
+    if (this.defaultFlowArgs.fileCollectColumns &&
+        this.defaultFlowArgs.fileCollectColumns.length > 0) {
+        this.fileCollectionSettingsShown = true;
+    }
+
     this.form.patchValue(this.defaultFlowArgs);
   }
 
+  openCollection() {
+    this.fileCollectionSettingsShown = true;
+  }
+
   openSettings() {
-    this.settingsShown = true;
+    this.lowLevelSettingsShown = true;
+  }
+
+  addFileCollectedColumn(event: MatChipInputEvent): void {
```

Comment:
Current naming is a bit inconsistent. We have `fileCollected`, `fileCollect` and `fileCollection` in this file. I'd settle on one of them (`fileCollection` being the easiest choice).